lib/Server.js

```diff
@@ -555,7 +555,7 @@ class Server {
 
     if (
       contentBase !== false &&
-      (options.serveIndex || typeof options.serveIndex === 'undefined')
+      (options.serveIndex || options.serveIndex === undefined)
```

Comment:
Just simplify:
```
const shouldHandleServeIndex = contentBase || options.serveIndex
```

`serveIndex` by default should be true

Reply:
@evilebottnawi i've simplified the condition & i added a comment to elaborate why i'm comparing with `undefined` 
```
          if (contentBase !== false) {
    // checking if it's set to true or not set (Default : undefined => true)
    options.serveIndex = options.serveIndex || options.serveIndex === undefined;

    const shouldHandleServeIndex = contentBase && options.serveIndex;

    if (shouldHandleServeIndex) {
      defaultFeatures.push('contentBaseIndex');
    }

```